src/init.cpp

```diff
@@ -432,6 +440,7 @@ void SetupServerArgs(NodeContext& node)
                  strprintf("Maintain an index of compact filters by block (default: %s, values: %s).", DEFAULT_BLOCKFILTERINDEX, ListBlockFilterTypes()) +
                  " If <type> is not supplied or if <type> = 1, indexes for all known types are enabled.",
                  ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-coinstatsindex", strprintf("Maintain coin statistics index, used by the gettxoutset rpc call (default: %u)", DEFAULT_COINSTATSINDEX), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
9981a8f

- s/rpc/RPC/ (can also fix the one in `txindex`)

- omit "call" (the C in RPC stands for "call")

- drop the comma at s/index, /index /

- nit: sort for both `-coinstatsindex` and `-blockfilterindex` (they should be at line 411)

Reply:
Fixed